### PR TITLE
update pack to install psr-18 client

### PR DESCRIPTION
**Pull request by dbu**
do we need to add something for the psr-17 factories or where do those come from?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,10 +12,10 @@
     }
   ],
   "require": {
-    "php-http/httplug-bundle": "^1.7",
-    "php-http/curl-client": "^1.7",
+    "php-http/httplug-bundle": "^2.0",
+    "php-http/curl-client": "^2.0",
     "php-http/message": "^1.0",
```
Comment (dbu) [inline composer.json:17]: do we need to bump the version of message too?

File: composer.json [hunk 2]
```diff
@@ -12,10 +12,10 @@
     }
   ],
   "require": {
-    "php-http/httplug-bundle": "^1.7",
-    "php-http/curl-client": "^1.7",
+    "php-http/httplug-bundle": "^2.0",
```
Comment (Nyholm) [inline composer.json]: No need for this

**Review by Nyholm — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Nyholm) [reply composer.json:17]: No, not really. Or not until we add the BC layer that Mark started on.

Comment (dbu) [reply composer.json:17]: cool

File: composer.json (hunk 2 above)
Comment (dbu) [reply composer.json]: dunno why i keep thinking we bumped the major version here... fixed.

**Comment by dbu** [comment]
we need to tag curl-client and client-common before we release this.

do we need to tag a release of the pack for flex to see it or does flex look at master?

**Comment by Nyholm** [comment]
Yes. We need to tag a release on this package. 

But I will still wait before I merge this.

**Comment by nicodemuz** [comment]
Bump.. @Nyholm can this be merged?

**Review by Nyholm — APPROVED** [review]

**Comment by Nyholm** [comment]
I tested and it does not work with guzzle/psr17. 

I added nyholm/psr7 because it supports psr17. 

Thank you @MrNicodemuz

**Comment by Nyholm** [comment]
And thank you @dbu

**Pull request merged**